relnotes/pkg-iteration.feature.md

```diff
@@ -0,0 +1,15 @@
+## The package iteration can now be specified via `PKGITERATION`
+
+* `Makd.mak`
+
+  Package versions consist of two components: the upstream version, and
+  the package iteration.  Previously this was hardcoded to match the
```

Comment:
Since the main supported packaging is Debian, I think it might be worth to clarify that *package iteration* in Debian lingo is the `debian_version`.

Reply:
`debian_revision`, not debian version (see: https://www.debian.org/doc/debian-policy/#version).  But yup, good call.